apps/atomy-q/API/app/Http/Controllers/Api/V1/RiskComplianceController.php

```
@@ -61,36 +47,38 @@ public function index(Request $request): JsonResponse
-            ],
-            'meta' => [
-                'rfq_id' => $rfqId,
-            ],
-        ]);
+        return response()->json($result);
     }
 
     /**
      * POST /risk-items/:id/escalate
      */
     public function escalate(Request $request, string $id): JsonResponse
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)